tests/src/dir.props

```diff
@@ -55,7 +55,7 @@
     <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
     <!-- Disable some C# warnings for the tests. -->
-    <NoWarn>78,162,164,168,169,219,251,252,414,429,642,649,652,675,1691,1717,1718,3001,3002,3003,3005,3008</NoWarn>
+    <NoWarn>78,162,164,168,169,219,251,252,414,429,618,642,649,652,675,1685,1691,1717,1718,3001,3002,3003,3005,3008</NoWarn>
```

Comment:
CS1685 sounds like RefAssemblies are misconfigured somewhere.

Reply:
`CS1685` was left due to the https://github.com/dotnet/corefx/issues/31395. As long as `benchmark.csproj` and `performance.csproj` are built against incompatible frameworks (`netstandard1.4` and `netstandard1.6` with `netcoreapp3.0` explicitly referenced assemblies) CS1685 will persist. Best solution would be to move this projects to `netstandard2.0` which is according to tools compatible with `netcoreapp3.0`